deploy/obs-csi-plugin/kubernetes/csi-obs-controller.yaml

```diff
@@ -0,0 +1,109 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: csi-obs-controller
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: csi-obs-controller
+  template:
+    metadata:
+      labels:
+        app: csi-obs-controller
+    spec:
+      serviceAccountName: csi-obs-controller-sa
+      nodeSelector:
+        kubernetes.io/os: linux
+      priorityClassName: system-cluster-critical
+      tolerations:
+        - key: "node-role.kubernetes.io/master"
+          operator: "Equal"
+          value: "true"
+          effect: "NoSchedule"
+      hostPID: true
+      containers:
```

Comment:
missing `cis-resizer` continer, pls add it

Reply:
done